.prettierrc

```diff
@@ -23,7 +25,26 @@
       }
     },
     {
-      "files": "./**/css/*.css",
+      "files": "*.{yml,yaml}",
+      "options": {
+        "parser": "yaml",
+        "tabWidth": 2,
+        "useTabs": false,
+        "semi": false,
+        "singleQuote": true
```

Comment:
should be false?

```suggestion
        "singleQuote": true
```
```suggestion
        "singleQuote": true
```

Reply:
for yaml files I believe single quotes are the preferred best practice. However with the change to ignoring embedded code altogether I'm not sure this matters much anymore.